tools/nanopolish/nanopolish_variants.xml

```diff
@@ -0,0 +1,85 @@
+<tool id="nanopolish_variants" name="Finds SNPs and polishes" version="0.1.0">
+    <macros>
+        <import>macros.xml</import>
+    </macros>
+    <expand macro="requirements" />
+    <command detect_errors="exit_code"><![CDATA[
+        ln -s '$input_merged' reads.fasta && 
+        ln -s '$input_reads_tar' fast5_files.tar.gz &&
+        tar -xzf fast5_files.tar.gz &&
```

Comment:
it would be more save to use -C here and explicitly unpack into a directory ....